contrib/pg_tde/documentation/docs/functions.md

```diff
@@ -201,12 +201,12 @@ Use these functions to create a new principal key for a specific scope such as a
 
 Princial keys are stored on key providers by the name specified in this function - for example, when using the Vault provider, after creating a key named "foo", a key named "foo" will be visible on the Vault server at the specified mount point.
 
-### pg_tde_set_principal_key
+### 
```

Comment:
You accidentally deleted this.

Reply:
Yep

BTW this PR is made on top of https://github.com/percona/postgres/pull/126
So it contains a lot of commits from there, please review 126 first

Reply:
Fixed